src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -612,8 +617,10 @@ private IEnumerable<CompletionItem> GetModulePathCompletions(SemanticModel model
                 entered = "";
             }
 
-            // These should only fail if we're not able to resolve cwd path or the entered string
-            if (TryGetFilesForPathCompletions(model.SourceFile.FileUri, entered) is not {} fileCompletionInfo)
+            // These should only fail if:
+            // 1. we're not able to resolve cwd path or the entered string
+            // 2. entered starts with br schema.
+            if (entered.StartsWith("br/", StringComparison.Ordinal) || entered.StartsWith("br:", StringComparison.Ordinal) || TryGetFilesForPathCompletions(model.SourceFile.FileUri, entered) is not {} fileCompletionInfo)
```

Comment:
>entered.StartsWith("br/", StringComparison.Ordinal)  [](http://example.com/codeflow?start=16&length=52)

If someone has a folder called `br` under their bicep file, won't this break path completions for that folder?

Reply:
@majastrz, fixed this issue and added tests:
https://github.com/Azure/bicep/commit/b6fc161b5344d6591c19ac0573f026b0eaddec25